build/rakelib/genesis.rake

```diff
@@ -14,19 +14,30 @@ namespace :genesis do
     end
 
     desc "Boot the new scaffolded network in docker-compose"
-    task :boot, [:chainnet] do |t, args|
-      if args[:chainnet] == nil
+    task :boot, [:chainnet, :eth_address, :eth_keys, :eth_websocket] do |t, args|
+      trap('SIGINT') { puts "Exiting..."; exit }
```

Comment:
whats the trap do?

Reply:
Just a way to exit a program gracefully, without ruby throwing errors when a `SIGINT` is sent to the process. Without it, you end up with something like the following:

```
rakelib/genesis.rake:83:in `system'
/rakelib/genesis.rake:83:in `build_docker_image'
build/rakelib/genesis.rake:39:in `block (3 levels) in <top (required)>'
```

Reply:
oh nice.